toolset/benchmark/benchmarker.py

```diff
@@ -358,30 +358,6 @@ def __run_test(self, test):
                     format(name=test.name))
                 return sys.exit(0)
 
-            out.write(
-                "test.os.lower() = {os}  test.database_os.lower() = {dbos}\n".
-                format(os=test.os.lower(), dbos=test.database_os.lower()))
-            out.write("self.results.frameworks != None: {val}\n".format(
-                val=str(self.results.frameworks != None)))
-            out.write("test.name: {name}\n".format(name=str(test.name)))
-            out.write("self.results.completed.: {completed}\n".format(
-                completed=str(self.results.completed)))
-            if self.results.frameworks != None and test.name in self.results.completed:
-                out.write(
-                    'Framework {name} found in latest saved data. Skipping.\n'.
-                    format(name=str(test.name)))
-                print(
-                    'WARNING: Test {test} exists in the results directory; this must be removed before running a new test.\n'.
-                    format(test=str(test.name)))
-                return sys.exit(1)
-            out.flush()
-
-            out.write(header("Beginning %s" % test.name, top='='))
-            out.flush()
-
-            # Start this test
-            out.write(header("Starting %s" % test.name))
-            out.flush()
```

Comment:
I don't think any of this code has happened for a very long time (since we got rid of the `latest` dir).